.github/workflows/tests.yml

```diff
@@ -181,6 +181,16 @@ jobs:
           COVERALLS_PARALLEL: true
         run: coveralls --service=github
 
+  towncrier-check: 
+    name: Verify Towncrier entry added
+    runs-on: ubuntu-latest
+    working-directory: /
```

Comment:
GitHub rejects `working-directory` here as invalid -- it would need to be inside the step (i.e. same level a `run:`) -- but I'm also moderately sure that the default working directory is already the folder that contains `pyproject.toml`, so I'm not sure what is happening here.